bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/discovery/BigAssFanDiscoveryService.java

```diff
@@ -242,22 +248,28 @@ private synchronized void deviceDiscovered(BigAssFanDevice device) {
                 .withRepresentationProperty(THING_PROPERTY_MAC).withLabel(device.getLabel()).build());
     }
 
-    private void schedulePollJob() {
-        logger.debug("Scheduling discovery poll job to run every {} seconds starting in {} sec", POLL_FREQ, POLL_DELAY);
-        cancelPollJob();
```

Comment:
Is the cancellation of any existing running instance intentional?

Reply:
I removed the active cancellation in the revised method. But now that you mention it, there might be more reasons to cancel it first and then start it. Looking from a distance this method was re-scheduling the task and not any scheduling it. So i changed the method to the old behaviour. 

On a side note, i don't think any user will ever notice this.